.github/workflows/sonarcloud-analysis.yml

```diff
@@ -52,9 +52,9 @@ jobs:
       - name: Set Common Sonar Variables
         id: sonar_env
         run: |
-          echo "##[set-output name=sonar_opts;]$(echo -Dsonar.host.url=https://sonarcloud.io \
+          echo "sonar_opts=-Dsonar.host.url=https://sonarcloud.io \
             -Dsonar.projectKey=com-pas_compas-scl-data-service \
-            -Dsonar.organization=com-pas )"
+            -Dsonar.organization=com-pas \" >> $GITHUB_OUTPUT
```

Comment:
There's a " missing/leftover

-Dsonar.organization=com-pas >> $GITHUB_OUTPUT

Reply:
You can absolutely modify my Pull Requests when you want if you are a maintainer (I allowed edits to maintainers) or pick my work and start a new branch and PR if not.